client/styles/main/_footer.scss

```diff
@@ -7,16 +7,27 @@ footer.navbar {
   color: $brand-bg-color;
   text-align: center;
   transition: all 0.5s ease 0s;
+  overflow: scroll;
```

Comment:
I'm getting a scroll bar on the footer in all browsers - shouldn't have any scrolling in the footer.

Reply:
Sorry, I had added it for an extremely minimized screen but forgot to test it normally lol